### PR TITLE
Yeoman-generator: add error function to base type

### PR DESCRIPTION
**Pull request by kyeotic**
this function is the recommended method for raising errors.
http://yeoman.io/environment/Environment.html#error
https://stackoverflow.com/questions/27616689/how-to-gracefully-abort-yeoman-generator-on-error

Please fill in this template.

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.

### CODE REVIEW

**Comment by tyrsius** [comment]
@typscript-bot done

**Comment by tyrsius** [comment]
@typescript-bot done

File: types/yeoman-generator/index.d.ts [hunk 1]
```diff
@@ -61,7 +61,9 @@ declare class Base extends EventEmitter {
 
     constructor(args: string|string[], options: {});
 
-    env: {};
+    env: {
```
Comment (janslow) [inline types/yeoman-generator/index.d.ts]: I think this type should be extracted as an `Environment` interface.

**Review by janslow — CHANGES_REQUESTED** [review]

File: types/yeoman-generator/index.d.ts (hunk 1 above)
Comment (andy-ms) [reply types/yeoman-generator/index.d.ts]: Eh, don't think it's necessary if there's only one member right now.

**Pull request merged**